sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -3,13 +3,15 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
 
+pub mod clients;
 mod generated;
+pub mod pipeline;
```

Comment:
The pipeline should not be public. This is an implementation detail. If you need it elsewhere in your crate, use `pub(crate)`.